plugins/services/src/js/components/ConfigurationMapTable.js

```diff
@@ -120,7 +139,20 @@ class ConfigurationMapTable extends React.Component {
       return column !== null;
     });
 
-    return <Table {...Object.assign({}, this.props, {columns})} />;
+    if (editLink) {
+      columns.push({
+        heading() { return null; },
+        className: 'configuration-map-action',
+        prop: 'edit',
+        render() { return editLink; }
+      });
```

Comment:
Since this is a "special" kind of column, could we change the property name to something that won't be used by the data, for example `_edit`?  And perhaps add `sortable: false`? Or is it enough to return `null` by the heading?